src/main/java/uk/nhs/hee/tis/trainee/sync/repository/PlacementSpecialtyRepository.java

```diff
@@ -2,20 +2,28 @@
 
 import java.util.Optional;
 import java.util.Set;
+import org.springframework.cache.annotation.CacheConfig;
+import org.springframework.cache.annotation.CacheEvict;
+import org.springframework.cache.annotation.CachePut;
+import org.springframework.cache.annotation.Cacheable;
 import org.springframework.data.mongodb.repository.MongoRepository;
 import org.springframework.data.mongodb.repository.Query;
 import org.springframework.stereotype.Repository;
 import uk.nhs.hee.tis.trainee.sync.model.PlacementSpecialty;
 
+@CacheConfig(cacheNames = PlacementSpecialty.ENTITY_NAME)
```

Comment:
Added PlacementSpecialty caching. May not be desired (since there are a lot of these!)

Reply:
Now removed